pkg/kubectl/describe.go

```diff
@@ -1587,7 +1587,7 @@ func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error)
 		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", d.CreationTimestamp.Time.Format(time.RFC1123Z))
 		fmt.Fprintf(out, "Labels:\t%s\n", labels.FormatLabels(d.Labels))
 		fmt.Fprintf(out, "Selector:\t%s\n", labels.FormatLabels(d.Spec.Selector))
-		fmt.Fprintf(out, "Replicas:\t%d updated / %d total\n", d.Status.UpdatedReplicas, d.Spec.Replicas)
+		fmt.Fprintf(out, "Replicas:\t%d updated / %d total; %d available, %d unavailable\n", d.Status.UpdatedReplicas, d.Spec.Replicas, d.Status.AvailableReplicas, d.Status.UnavailableReplicas)
```

Comment:
Let's have this as `Replicas:\t%d updated | %d total | %d available | %d unavailable\n`. I don't feel strong on the choice of the separator but I want it to be the same everywhere in the field.